roles/ucs/defaults/main.yml

```diff
@@ -9,24 +9,27 @@ operator_group: "{{ operator_user }}"
 # docker
 
 docker_registry: index.docker.io
-docker_registry_ucs: "{{ docker_registry }}"
+docker_registry_ucs: docker-test.software-univention.de
```

Comment:
The image should come from a semi-official registry (Docker Hub, Quay, ..). Is this not published anywhere?

Reply:
As the commit message suggests, this temporary solution required adjustments to the specifics of the testbed (MTU, Port-Number, HTTPS termination at traefik) which are not useful for other purposes. The source code of the container is published under https://github.com/univention/ucs-appliance-container and we will also publish that on docker hub soon. But for this project we currently need the adjustments specific for the testbed.